cmd/kubeadm/app/cmd/init.go

```diff
@@ -51,7 +52,11 @@ const (
 
 var (
 	initDoneMsgf = dedent.Dedent(`
-		Kubernetes master initialised successfully!
+		Your Kubernetes master has initialized successfully!
+
+		But you still need to deploy a pod network to the cluster.
+		You should "kubectl apply -f" some pod network yaml file that's listed at:
```

Comment:
Suggest:

You should now deploy a pod network to the cluster. 
Run "kubectl apply -f [podnetwork].yaml" with one of the options listed at: